include/open62541/plugin/certificategroup.h

```diff
@@ -40,6 +40,31 @@ struct UA_CertificateGroup {
     void (*clear)(UA_CertificateGroup *certGroup);
 };
 
+struct UA_Transaction;
+typedef struct UA_Transaction UA_Transaction;
+
+struct UA_Transaction {
+    UA_NodeId sessionId;
+
+    size_t certGroupSize;
+    UA_CertificateGroup *certGroups;
```

Comment:
Are arbitrary certificate groups supported here or is this restricted to DefaultApplicationGroup and DefaultUserTokenGroup?